ArkKit/Ark.swift

```diff
@@ -12,6 +12,9 @@ import Foundation
 class Ark {
     let rootView: any AbstractRootView
     let arkState: ArkState
+    var gameLoop: AbstractArkSimulator?
```

Comment:
I'm still figuring out if this is the best place to initialise the `gameLoop`... give me some time to ponder

Reply:
Lets KIV this for now